sqs.tf

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (C) 2019 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ */
+
+resource "aws_sqs_queue" "shuntingyard_sqs_queue" {
+  name = "${local.instance_alias}-sqs-queue"
+}
+
+resource "aws_sqs_queue_policy" "shuntingyard_sqs_queue_policy" {
+  queue_url = "${aws_sqs_queue.shuntingyard_sqs_queue.id}"
+
+  policy = <<POLICY
+{
+  "Version": "2012-10-17",
```

Comment:
Thanks for adding this. I think it was working for me earlier coz this permission was added manually on the UI.